### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by talexandru1987**

### CODE REVIEW

File: src/utils/utils.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
+//select all from a table
+const selectAll = (aTable) => `SELECT * FROM ${aTable}`;
```
Comment (chelseanicholls) [inline src/utils/utils.js]: The variable names in this file make the functions quite difficult to read. For example, this variable could just be `table`.

File: src/utils/utils.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+//add department
+const addAdepartment = (anId, aName) =>
+  `INSERT INTO department (id, name) VALUES ('${anId}', '${aName}')`;
+
+// get max id
+const maxId = (aDepartment) => `SELECT max(id) as max FROM ${aDepartment}`;
+
+//add role
+const addArole = (anId, aTitle, aSalary, aDepartment_id) =>
```
Comment (chelseanicholls) [inline src/utils/utils.js]: Same as above, you don't need `an` and `a` at the beginning of the variable names. It is much easier to read if they are just `id`, `title`, etc.

File: src/utils/utils.js (hunk 1 above)
Comment (talexandru1987) [reply src/utils/utils.js]: thank you for the feedback I have amended the code

File: src/utils/utils.js (hunk 2 above)
Comment (talexandru1987) [reply src/utils/utils.js]: Done, thank you

**Pull request merged**